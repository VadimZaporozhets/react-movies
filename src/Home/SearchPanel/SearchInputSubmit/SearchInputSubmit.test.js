import React from 'react';
import { SearchInputSubmitComponent } from './index';
import { shallow } from 'enzyme';
import { SEARCH_BY_PARAMS } from '../../../constants';

describe('<SearchInputSubmitComponent />', () => {
    it('should render search component', () => {
        const props = {
            classes: expect.any(Object),
            onSubmit: jest.fn(),
            searchBy: SEARCH_BY_PARAMS.title
        };

        const wrapper = shallow(<SearchInputSubmitComponent {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
