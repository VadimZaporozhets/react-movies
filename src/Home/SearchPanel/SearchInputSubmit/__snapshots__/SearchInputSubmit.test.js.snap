// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchInputSubmitComponent /> should render search component 1`] = `
<WithStyles(Grid)
  container={true}
  spacing={40}
>
  <WithStyles(Grid)
    item={true}
    lg={10}
    md={8}
    sm={12}
  >
    <TextField
      fullWidth={true}
      placeholder="Search"
      required={false}
      select={false}
      variant="standard"
    />
  </WithStyles(Grid)>
  <WithStyles(Grid)
    item={true}
    lg={2}
    md={4}
    sm={12}
  >
    <WithStyles(Button)
      color="primary"
      variant="contained"
    >
      Search
    </WithStyles(Button)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
